src/views/HomeView.js

```diff
@@ -0,0 +1,204 @@
+          cards: personCards,
+        },
+        {
+          type: "custom:mushroom-template-card",
+          primary: "{% set time = now().hour %} {% if (time >= 18) %} Good Evening, {{user}}! {% elif (time >= 12) %} Good Afternoon, {{user}}! {% elif (time >= 5) %} Good Morning, {{user}}! {% else %} Hello, {{user}}! {% endif %}",
+          icon: "mdi:hand-wave",
+          icon_color: "orange",
+          tap_action: {
+            action: "none",
+          },
+          double_tap_action: {
+            action: "none",
+          },
+          hold_action: {
+            action: "none",
+          },
+        },
+      ];
+
+      // Add quick access cards.
+      if (options.quick_access_cards) {
+        homeViewCards.push(...options.quick_access_cards);
+      }
+
+      // Add area cards.
+      homeViewCards.push({
+            type: "custom:mushroom-title-card",
+            title: "Areas",
+          },
+          {
+            type: "vertical-stack",
+            cards: areaCards,
+          });
+
+      // Add custom cards.
+      if (options.extra_cards) {
+        homeViewCards.push(...options.extra_cards);
+      }
+
+      return homeViewCards;
+    });
+  }
+
+  /**
+   * Create the chips to include in the view.
+   *
+   * @return {Object[]} A chip object array.
+   */
+  async #createChips() {
+    const chips           = [];
+    const chipOptions     = Helper.strategyOptions.chips;
+    // TODO: Get domains from config (Currently strategy.options.views).
+    const exposed_domains = ["light", "fan", "cover", "switch", "climate"];
+    // Create a list of area-ids, used for switching all devices via chips
+    const areaIds         = Helper.areas.map(area => area.area_id);
+
+    let chipModule;
+
+    // Weather chip.
+    const weatherEntityId = chipOptions?.weather_entity ?? Helper.entities.find(
+        entity => entity.entity_id.startsWith("weather.") && entity.disabled_by == null && entity.hidden_by == null,
+    ).entity_id;
+
+    if (weatherEntityId) {
+      try {
+        chipModule        = await import("../chips/WeatherChip");
+        const weatherChip = new chipModule.WeatherChip(weatherEntityId);
+        chips.push(weatherChip.getChip());
+      } catch (e) {
+        console.error(Helper.debug ? e : "An error occurred while creating the weather chip!");
+      }
+    }
+
+    // Numeric chips.
+    for (let chipType of exposed_domains) {
+      if (chipOptions?.[`${chipType}_count`] ?? true) {
+        const className = Helper.sanitizeClassName(chipType + "Chip");
+        try {
+          chipModule = await import((`../chips/${className}`));
+          const chip = new chipModule[className](areaIds);
+          chips.push(chip.getChip());
+        } catch (e) {
+          console.error(Helper.debug ? e : `An error occurred while creating the ${chipType} chip!`);
+        }
+      }
+    }
+
+    // Extra chips.
+    if (chipOptions?.extra_chips) {
+      chips.push(chipOptions.extra_chips);
```

Comment:
What would be the example config if I add multiple extra chips?
Currently, I can only add 1 extra chip with the following config:
```YAML
    chips:
      extra_chips:
        type: entity
        entity: sensor.fajr_prayer
        content_info: state
        icon: mdi:weather-sunset-up
```

Reply:
AFAIA, I didn't change anything of the chip configuration and should still be the same as in the readme.

You might be missing a dash (-) at the line after `extra chips`, since `extra_chips` is supposed to be an array.

```yaml
    chips:
      extra_chips:
        - type: entity
          entity: sensor.fajr_prayer
          content_info: state
          icon: mdi:weather-sunset-up
        - type: conditional
          conditions:
            - entity: lock.front_door
              state: unlocked
          chip:
            type: entity
            entity: lock.front_door
            icon_color: red
            content_info: none
            tap_action:
              action: toggle
```

Reply:
Fixed in next commit